efel/io.py

```diff
@@ -103,6 +103,7 @@ def load_neo_file (file_name, stim_start=None, stim_end=None):
             returned object : [Segments_1, Segments_2, ..., Segments_n] 
             Segments_1 = [Traces_1, Traces_2, ..., Traces_n] 
     """
+
```

Comment:
After consideration, there is no way we could make it usefull for Neo, it is quite specificly made for eFEL. Thus I will split it up as requested to increase the readability.